src/gui/screen_menu_odometer.cpp

```diff
@@ -1,53 +1,29 @@
-// screen_menu_odometer.cpp
+/**
+ * @file screen_menu_odometer.cpp
+ */
 
-#include <stdlib.h>
-
-#include "cmath_ext.h"
-#include "screen_menus.hpp"
+#include "screen_menu_odometer.hpp"
 #include "screen_menu.hpp"
-#include "i18n.h"
 #include "odometer.hpp"
 #include "MItem_tools.hpp"
 #include "DialogMoveZ.hpp"
 
-using MenuContainer = WinMenuContainer<MI_RETURN>;
-
-using OdometerScreen = ScreenMenu<EFooter::On, MI_RETURN, MI_ODOMETER_DIST_X, MI_ODOMETER_DIST_Y, MI_ODOMETER_DIST_Z, MI_ODOMETER_DIST_E, MI_ODOMETER_TIME>;
-
-class ScreenOdometer : public OdometerScreen {
-    static constexpr const char *label = N_("ODOMETER");
-
-    static float getVal(Odometer_s::axis_t axis) {
-        Odometer_s::instance().force_to_eeprom();
-        return Odometer_s::instance().get(axis) * .001f;
-    }
-    static uint32_t getTime() {
-        Odometer_s::instance().force_to_eeprom();
-        return Odometer_s::instance().get_time();
-    }
```

Comment:
`getVal` and `getTime` methods disappeared. Is this change intended?

Reply:
it is unused, even in private

Reply:
fixed